apps/web/src/constants/auth.ts

```diff
@@ -27,6 +27,15 @@ export enum Roles {
   ContactManager = 2,
 }
 
+export enum OrganisationRoleNames {
+  ClinicalResearchSponsor = 'Clinical Research Sponsor',
+}
+
+export enum Wso2GroupsOperations {
```

Comment:
Minor: should the name be pluralised twice?
Wso2GroupsOperations -> Wso2GroupOperations ?

Reply:
I thought it would be fine, as these are the operations for the wso2 attribute called groups. Happy to change if you find it will be more clear as `Wso2GroupOperations`?

Reply:
The enum itself should always be singular (the last 's' - it'll make more sense when you use the enum as a parameter type).

We're also not consistent on using Role vs Group for the same concept.

I would go with `RoleOperation`.